textBasedAdventure.js

```diff
@@ -1,5 +1,439 @@
+        readline.question("If you can swim feel free to hop out now and swim the rest of the of way... Other wise your task will be to bring me a bacon egg and cheese sandwhich! Will you accept (yes || no)? ").toLowerCase();
+    }
+
+    if (dragonRequest === "yes") {
+        console.log("I'm in a good mood and can wait until you're done with you're race... Don't have me waiting for very long. I know your scent now I'll find you!");
+        underBridge()
+    } else if (dragonRequest === "no") {
+        console.log("In fury the dragon unleashes fire burning you and the boat to a crisp!");
+        startOver();
+    }
+}
+
+
+const bathroomSprint = () => {
+    console.clear();
+
+let findStall = readline.question("You've continuned on your route and your stomache starts to bubble, you see a public bathroom coming up. \n It looks and smell disgusting! But any second longer and you may have an accident!!! \n There are four stalls, which do you choose (1, 2 || 3) please enter a number? ");
+
+if (findStall !== Number) {
+    console.log("Please enter a number (1, 2 || 3)!")//What else do I need to put here? to make sure it runs again? or is aloop even okay here
+}
+let stalls = [1, 2, 3, 4]
+for (let i = 0; i < stalls.length; i++) {
+
+    if (stalls[i] === 1) {
+        console.log("You chose stall 1. Just your luck theres no tissue! You lose 40 minuets trying to find some!")
+        time -= .20
+    } else if (stalls[i] === 2) {
+        console.log("The stall you chose has no toilet tissue. You have to wait until someone comes to pass you tissue! You lost 30 mins")
+        GSP
+        time -= .30;
+    } else if (stalls[i] === 3) {
+        console.log("You chose well my friend. You handled your buisness, you only lost 15 minuets")
+        GSP 
+        time -= .15
+    }
+    underBridge()
+}
+
+}
+
+
+const talkingSpider = () => {
+    console.clear();
+
+    console.log("You stop and notice the spider. \n Spider: 'You dare come into my home causing a ruckus! Now I haven't had any visitors in a while so I want to do something nice for you.'");
+
+    let userAge = readline.question("Go on head tell me how old you are:(enter number for age) ");
+
+
+    if (userAge <= 10) {
+        console.log("Get a few more years on you and I'll give you something nice")
+    } else if (userAge <= 20) {
+        console.log("You've been around for while, here's some extra time! Good luck with the race!")
+        time += .40;
+    } else if (userAge <= 30) {
+        console.log(`Wow that's really something ${56} years old. I have faith in people and if you've been around for that long I'm sure you've lent a helping hand here and there. Here's some GSP! Good luck on the race!`)
+        GSP += userAge//convert user input "string to number and add that to the GSP"
+    } else if (userAge <= 40) {
+        console.log("something")
+    } else if (userAge <= 100) {
+        console.log(`${56} years whoo, thats really something! Here's some loving`)
```

Comment:
a valid way to use string interpolation here would have been if you added a variable instead of the number 56 since at the end it would all still be evaluated as a string.